src/secretsManager.ts

```diff
@@ -0,0 +1,36 @@
+import AWS from 'aws-sdk';
```

Comment:
Add the Copyright License Header on this file

Reply:
added